howto/update/upgrade-appliance.md

```diff
@@ -1,4 +1,4 @@
-Before you upgrade the Anbox Cloud Appliance, you should make sure all packages on the machines that are part of the deployment are up-to-date. To do so, run the following commands on each machine:
+Before you upgrade the Anbox Cloud Appliance, make sure all packages on the machines that are part of the deployment, are up-to-date. To do so, run the following commands on each machine:
```

Comment:
Why did you add a comma there? It looks weird to me, so I'm not sure if it's intended.

Reply:
It was a typo, thanks!